src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 0.3s;
```

Comment:
same as previous transition: better indicate transition-name, or tipe it transition: trans-name trans-dur trans-func delay..
in your case it can be "transition: all 0.3s;" "transition: color 0.3s" - second is better to avoid mistakes